deploy/scripts/deploy_quadlets.sh

```
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+source deploy/scripts/env.sh
+
+echo "Starting Deployment"
+
+export PRIMARY_IP=$(bash -c 'source ./test/scripts/functions && get_ext_ip')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)